package-system/AWSNativeSDK/Dockerfile.ubuntu.2004

```diff
@@ -0,0 +1,49 @@
+#
+# Copyright (c) Contributors to the Open 3D Engine Project.
+# For complete copyright and license terms please see the LICENSE at the root of this distribution.
+#
+# SPDX-License-Identifier: Apache-2.0 OR MIT
+#
+
+# This docker file uses ubuntu 20.04 as the base image so that the AWS Native C++ libraries will use OpenSSL 3 as the base
+# for its dependencies
+#
+
+FROM ubuntu:20.04
```

Comment:
(discussed offline) since this is to be used with the AWS SDK, suggest using a public ECR repository for this and the other dockerfile base image.
This verified Ubuntu repo is probably the right one to pull from: https://gallery.ecr.aws/ubuntu/ubuntu